web/style/cart-modal.scss

```diff
@@ -0,0 +1,139 @@
+// Cart Modal
+
+/* The Modal (background) */
+.modal {
+  display: none; /* Hidden by default */
+  position: fixed; /* Stay in place */
+  z-index: 1; /* Sit on top */
+  margin-top: 100px; /* Location of the box */
+  left: 49%;
+  top: 0;
+  width: 500px; /* Full width */
+  overflow: auto; /* Enable scroll if needed */
+  background-color: rgb(0, 0, 0); /* Fallback color */
+  background-color: rgba(0, 0, 0, 0.4); /* Black w/ opacity */
+  color: #fff;
+}
+
+/* Modal Content */
+.modal-content {
+  height: 80vh; /* Full height */
+  position: relative;
+  background-color: #eee;
+  margin: auto;
+  padding: 0;
+  border: 1px solid #888;
+  width: 100%;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+  animation-name: animatetop;
+  animation-duration: 0.4s;
+  color: #111;
+}
+
+.modal-header {
+  padding: 2px 16px;
+  background-color: $text-color-primary;
+  color: $background-color-primary;
+}
+
+.modal-body {
+  padding: 2px 16px;
+}
+
+.modal-footer {
+  padding: 2px 16px;
+  background-color: $background-color-secondary;
+  color: $background-color-primary;
+  bottom: 0;
+  position: absolute;
+  width: 100%;
+  text-align: center;
+  padding: 2px 16px;
+}
+
+.cart-item {
+  @extend .flex-row;
+  margin-top: 10px;
+  background-color: #fff;
+  &__img {
+    width: 20%;
+    height: 21%;
+  }
+}
+
+.cart-item-detail {
+  @extend .flex-column;
+  &__heading {
+    font-size: 1.5rem;
+    padding: 15px;
+  }
+  &__info {
+    @extend .flex-row;
+    font-size: 1rem;
+    align-items: center;
+
+    &--count {
+      @extend .flex-row;
+    }
+    &--item {
+      &-count {
+        margin-top: 4px;
+      }
+    }
+    &--unitPrice {
+      margin-top: 4px;
+    }
+    &--symbol {
+      margin-top: 2px;
+      font-size: 1.5rem;
+    }
+    &--remove,
+    &--add {
+      @include break-point(desktop) {

```

Comment:
This is wrong the media queries for a module should be independent blocks and should not be placed within mobile selectors. 
Secondly, Tablet view port CSS will come first and Desktop later. Mobile > Tablet > Desktop.